spec/models/embed/was_time_map_spec.rb

```diff
@@ -31,6 +31,32 @@
       end
     end
   end
+  describe '#timemap (new behavior)' do
+    context 'when HTTP is successful' do
+      before do
+        expect(Faraday).to receive(:get).and_return(double('response', body: timemap_new, success?: true))
+      end
+
+      it 'requests and parses timemap to an array' do
+        expect(subject.timemap.length).to eq 10
+        subject.timemap.each do |memento_line|
+          expect(memento_line).to be_an(Embed::WasTimeMap::MementoLine)
+        end
+      end
+
+      it 'the memento lines are not all memento entries' do
+        expect(subject.timemap.count(&:memento?)).to eq 7
+      end
+
+      it 'memento lines have accessors' do
+        memento_line = subject.timemap.find(&:memento?)
+        expect(memento_line.url).to eq 'https://swap.stanford.edu/20090718213431/http://ennejah.info/'
+        # NOTE: this needed changing to pass! Problematic?
+        expect(memento_line.rel).to eq 'memento'
```

Comment:
Heads-up. Not sure sul-embed otherwise special-cases "last memento" or "first memento"? if so, that's something that may trip us up.